src/options.json

```diff
@@ -4,6 +4,9 @@
     "name": {
       "type": "string"
     },
+    "mode": {
+      "type": "string"
```

Comment:
Can't we specify enumeration here?

Reply:
Yep, I think so will do (need to look it up again first 😛 ). Otherwise needs to throw in the code